docs/openapi_generator/strong_typing/schema.py

```diff
@@ -531,6 +531,7 @@ def type_to_schema(self, data_type: TypeLike, force_expand: bool = False) -> Sch
             # add property docstring if available
             property_doc = property_docstrings.get(property_name)
             if property_doc:
+                # print(output_name, property_doc)
```

Comment:
remove?

Reply:
yes, I kept it on purpose as a reminder about where I had added this print. It took me a long time yesterday to figure out this doc stuff :)